resources/views/partials/common/preloader.blade.php

```diff
@@ -0,0 +1,9 @@
+@inject('layoutHelper', 'JeroenNoten\LaravelAdminLte\Helpers\LayoutHelper')
```

Comment:
This line is not needed, since `LayoutHelper` is not used inside this blade file